packages/shared/src/hooks/profile/useDevCard.ts

```diff
@@ -27,7 +27,10 @@ export interface UseDevCard {
   coverImage: string;
 }
 
-export const useDevCard = (userId: string): UseDevCard => {
+export const useDevCard = (
+  userId: string,
+  onSuccess?: (data: DevCardData) => void,
+): UseDevCard => {
```

Comment:
We use object syntax for hook params eg. `{ userId }` to allow new params to be added without reordering or updating.